lib/std/unicode.zig

```diff
@@ -770,6 +770,40 @@ fn calcUtf16LeLen(utf8: []const u8) usize {
     return dest_len;
 }
 
+/// Print the given `utf16le` string
+fn formatUtf16le(
+    utf16le: []const u16,
+    comptime fmt: []const u8,
+    options: std.fmt.FormatOptions,
+    writer: anytype,
+) !void {
+    _ = fmt;
+    _ = options;
+    var buf: [300]u8 = undefined; // just a random size I chose
+    var it = std.unicode.Utf16LeIterator.init(utf16le);
+    var u8len: usize = 0;
```

Comment:
Nope, the errors must be caught and handled. I'd suggest printing U+FFFD in case of error.

Reply:
What about when the `utfEncode` fails, same thing, print `U+FFFD`?

Reply:
I updated PR to print `U+FFFD` for all decoding/encoding errors.